bin/hdfcoinc/pycbc_fit_sngls_by_template

```diff
@@ -133,26 +145,23 @@ tb_hashorder = tb[hash_sort]
 # reorder template IDs in parallel to the boundary values
 tid_hashorder = tid[hash_sort]
 
-# Calculate the differences between the boundary indices to get the 
+# Calculate the differences between the boundary indices to get the
 # number in each template
 # adding on total number at the end to get number in the last template
-total_number = len(trigf[args.ifo + '/template_id'][:])
+total_number = len(trigf[args.ifo + '/template_id'])
```

Comment:
I found the following on h5py docs: 
> On 32-bit platforms, len(dataset) will fail if the first axis is bigger than 2**32. It’s recommended to use Dataset.len() for large datasets 
Not at 2**32 yet, but we can't rule it out ..

Reply:
Doesn't really matter actually. We can't even build pycbc on a purely 32bit platform. We require vector instructions that have only existed since the over a decade move to 64 bits. I think we are safe here.